packages/modeling/src/color/color.js

```diff
@@ -1,10 +1,37 @@
 const flatten = require('../utils/flatten')
 
+const geom2 = require('../geometry/geom2')
+const geom3 = require('../geometry/geom3')
+const path2 = require('../geometry/path2')
+const poly3 = require('../geometry/poly3')
+
+const colorGeom2 = (color, object) => {
+  const newgeom2 = geom2.create(geom2.toSides(object))
```

Comment:
would it not make more sense to just use the clone() function ?

Reply:
Alas, clone is a DEEP clone, which makes full copies of all polygons, and vertices. Very expensive.

Reply:
Actually, thinking a little more about this. The clone functions should be SHALLOW copies, as geometry is 'immutible' by design. But there may be some side effects of changing the clone functions.

@kaosat-dev do you think this is worth correcting now?

Reply:
@z3dev sorry for the slow reply. This has been nagging me for a few days, I am a bit baffled, but I agree this needs a closer attention to the clone() functions : what do you mean by 'deep clone' ? we just want a fresh copy of the data, and that already seems to be the case (I looked at Geom2.clone(), Geom3.clone() etc)

Reply:
DEEP CLONE.... all data is replicated... new arrays

SHALLOW CLONE... the polygons and the transforms are passed on...

Reply:
I create issue #573 to debate this further.